### ISSUE TITLE
PDF: Refine layout

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/metanorma-iho#74]
Source tasks: https://github.com/metanorma/mn-samples-iho/issues/76, https://github.com/metanorma/metanorma-iho/issues/72

> IHO layouts need some updating and clean up to the latest best practices.

**Comment by Intelligent2013** [comment]
`metanorma-iho` updated in https://github.com/metanorma/metanorma-iho/pull/85 for:
- `Table of Contents` changed to `Contents`
- ` 4b quai Antoine 1er` changed to ` 4b quai Antoine 1<sup>er</sup>` for cover page text:
![image](https://user-images.githubusercontent.com/5559926/202003809-fe7ccbc2-ac38-48d3-a6bd-6126acc0c602.png)

- font `Calibri` added for rendering of the quote text in `license-statement`:
![image](https://user-images.githubusercontent.com/5559926/202003907-67bd31bc-93e5-4690-8fd3-76dca1ea0cf7.png)

**Comment by Intelligent2013** [comment]
- there aren't dots at the end of clause's numbers in the original PDF:
![image](https://user-images.githubusercontent.com/5559926/202217694-0bb1912d-13bf-4cbd-bf29-bf58fe7edeef.png)

Rendered PDF:
![image](https://user-images.githubusercontent.com/5559926/202218100-b8d64653-f30d-4667-9ae3-260efdb56106.png)

- Annex letter and title on the one line:
![image](https://user-images.githubusercontent.com/5559926/202218542-7c770583-1635-4d47-9a0d-2ad1c7c0834c.png)

Rendered PDF:
![image](https://user-images.githubusercontent.com/5559926/202218604-c382d51e-fcff-4285-9316-13b646f6b320.png)

Note: in the original S-44 there isn't dot after 'Annex':
![image](https://user-images.githubusercontent.com/5559926/202219032-a96b6f39-0189-4b7b-a105-a36ef3d799ba.png)

**Comment by opoudjis** [comment]
@Intelligent2013 I have nothing but guilty admiration about the fact that you end up doing a lot of work that I don't have the time to do...

... except, there are some instances where the problem really does need to be resolved in Presentation XML, because the solution needs to be reflected in HTML and DOC output. Full stops are clause numbers are arguably such a case. 

Note that in Presentation XML (because this is a recurring issue), there is a function `clausedelim` which outputs the clause delimiter, if any. This is the line in isodoc that defines how titles are altered in clauses in Presentation XML:

`prefix_name(elem, "<tab/>", "#{lbl}#{clausedelim}", "title")`

In isodoc, clausedelim returns "." . In metanorma-itu, it is redefined to "", since ITU does not have dots.

So.... if you want there to be no dots in IHO, then clausedelim in metanorma-iho needs to be redefined to return "".

I do have to raise the concern though about the inconsistencies of IHO formatting that we have seen in recent documents. Are we certain that the dot should be suppressed for all clauses and annexes in all of IHO?

**Comment by ronaldtse** [comment]
@opoudjis @Intelligent2013 can you create tickets for questions you want clarification from IHO? I can't keep track of questions inserted into answers.

**Comment by Intelligent2013** [comment]
Currently, IHO XSLT adapted for compliance PDF with S-102, except Clauses/Annex numbering format (https://github.com/metanorma/metanorma-iho/issues/74#issuecomment-1317178587).
Need update after resolving https://github.com/metanorma/metanorma-iho/issues/88.
On hold.

**Comment by opoudjis** [comment]
@Intelligent2013 Is the dot after clause numbers issue dependent on #88?

**Comment by Intelligent2013** [comment]
> @Intelligent2013 Is the dot after clause numbers issue dependent on #88?

@opoudjis no, all documents don't have dot after clause numbers. So regarding 

> Are we certain that the dot should be suppressed for all clauses and annexes in all of IHO?

I think that the dot should be suppressed for all clauses. But I don't sure about dot after `Annex` at 1st level title.

**Comment by opoudjis** [comment]
Pending review from IHO secretariat, Jeff Woden; if secretariat happy, will leave as is.

**Comment by JeffWootton** [comment]
There should be no full stop/period ("dot") after any clause numbers in IHO Publications. This is the convention that I have applied to all IHO Publications that I have had the opportunity to review the format since I commenced as the TSSL at the IHO Secretariat.

The same applies to Annexes.

Happy to discuss further as required.
